packages/ui/lib/components/Detail/Detail.tsx

```diff
@@ -101,14 +101,14 @@ export const Detail = ({
         children !== undefined &&
         (typeof label === 'string' && tooltip ? (
           <Tooltip tip={children} label={label} side="bottom">
-            <div className="flex items-center gap-2 text-right cursor-pointer">
+            <div className="flex items-center gap-2 cursor-pointer">
               <Value truncate={truncate} size={valueSize}>
                 {children}
               </Value>
             </div>
           </Tooltip>
         ) : (
-          <div className="flex items-center gap-2 text-right">
```

Comment:
`text-right` in unnecessary and cause alignment issue

**Issue**
 
<img width="400" alt="Screenshot 2023-05-25 at 14 17 47" src="https://github.com/unlock-protocol/unlock/assets/20865711/d8231476-8316-491b-bfe2-b2bb5a393f0c">

**Fix** 

<img width="400" alt="Screenshot 2023-05-25 at 14 18 02" src="https://github.com/unlock-protocol/unlock/assets/20865711/1e48eef9-7c82-4945-910f-e2ccd66b9008">